data/wanderers.txt

```diff
@@ -425,6 +427,64 @@ phrase "friendly wanderer"
 	word
 		"."
 			
+phrase "friendly wanderer"
+	word
+		"Wanderer "
+		"Our "
+		"These "
+	word
+		"worlds"
+		"planets"
+		"places"
+		"lands"
+		"ecosystems"
+		"biospheres"
+	word
+		" are open to "
+		" are available to "
+		" welcome "
+	word
+		"anyone"
+		"anybody"
+		"any being"
+		"any lifeform"
+		"any [identity? consciousness? entity?]"
+	word
+		" "
+		", if they are "
+		", so long as they are "
+		", as long as they are "
+		", [providing, given] they are "
+	word
+		"willing"
+		"wishing"
+		"wanting"
+		"intending"
+		"looking"
+		"meaning"
+		"planning"
+	word
+		" to "
+	word
+		"help"
+		"assist"
```

Comment:
"Assist" doesn't work here - it'll lead to phrases like "assist us restore," etc. Instead you could choose between "help," "help us to," and "assist us to."